src/main.cpp

```diff
@@ -23,7 +23,8 @@ static int protothreadReadDHT(struct pt *pt)
     PT_WAIT_UNTIL(pt, millis() - lastTimeread > interval);
     float temp = readtemp();
     float humid = readhumidity(); 
-    Serial.printf("temp: %.3f, humid: %.3f\n ", temp, humid);
+    Serial.printf("temp: %.3f, humid: %.3f\n", temp, humid);
```

Comment:
why not use ets_printf() function here too?

Reply:
**Read** the documentation dear, ``` C
/**
  * @brief  Printf the strings to uart or other devices, similar with printf, simple than printf.
  *  Can not print float point data format, or longlong data format.
  *  So we maybe only use this in ROM.
  *
  * @param  const char *fmt : See printf.
  *
  * @param  ... : See printf.
  *
  * @return int : the length printed to the output device.
  */
int ets_printf(const char *fmt, ...);```

Reply:
Noted. Thanks